### ISSUE TITLE
Non-fatal molgenis-core-ui build errors

### ISSUE THREAD

**Issue by dennishendriksen** [molgenis/molgenis#5951]
### How to Reproduce
- Run ```mvn clean verify```
- Inspect logs

### Expected behavior
- No lines marked with [ERROR] in molgenis-core-ui build

### Observed behavior
```[INFO] Running 'npm install --color=false' in /data/jenkins/git/molgenis/molgenis-core-ui
[ERROR] npm WARN deprecated babel-plugin-module-alias@1.6.0: WARNING: This project has been renamed to babel-plugin-module-resolver. Install babel-plugin-module-resolver for new features
[ERROR] npm WARN optional dep failed, continuing fsevents@1.1.1
[ERROR] npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[ERROR] npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[ERROR] npm WARN deprecated lodash@1.0.2: lodash@<2.0.0 is no longer maintained. Upgrade to lodash@^3.0.0
[ERROR] npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs
' to find it in the tree.
[ERROR] npm WARN optional dep failed, continuing fsevents@1.1.1```

**Comment by mswertz** [comment]
we should make sure log only contains real errors, otherwise we will start to ignore them

**Comment by dennishendriksen** [comment]
Fixed by #8201
